### PR TITLE
Switched to Flask endpoint | Logic for test | Added logic for multiple languages

### PR DESCRIPTION
**Pull request by rjuliang**
Changed endpoint to hit the Flask integration instead of Nodejs,
Tested
Created test file requested by Dr. Scannell
Adjusted CORS in Flask file
Included multiple language support in spellCheck file

### CODE REVIEW

File: Flask-Integration/spellCheck.py [hunk 1]
```diff
@@ -13,27 +13,32 @@
 
 def words(text): return re.findall(r'\w+', text)
 
-WORDS = Counter(words(open('bigText.txt').read()))
+WORDS_ENGLISH = Counter(words(open('bigText.txt', encoding="UTF-8").read()))
+WORDS_IRISH = Counter(words(open('legal.txt', encoding="UTF-8").read() + open('bible.txt', encoding="UTF-8").read() + open('news.txt', encoding="UTF-8").read() ))
```
Comment (kimaya1008) [inline Flask-Integration/spellCheck.py:17]: We may need to review the logic to switch languages.

**Review by kimaya1008 — COMMENTED** [review]
We need a comment to explain the Irish files are not added to the repo.

File: Flask-Integration/get-localhost.py [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+import json
 
 #export FLASK_APP=get-localhost.py
 #python -m flask run
 
 app = Flask(__name__)
 
+cors = CORS(app)
+app.config['CORS_HEADERS'] = 'Content-Type'
+
```
Comment (kimaya1008) [inline Flask-Integration/get-localhost.py:13]: This is a little difficult to understand, we'd just get together once to understand this better.

**Review by kimaya1008 — APPROVED** [review]
For the Beta version, everything looks good.
However, the stated changes can be done later.
Hence, approving.

File: Flask-Integration/spellCheck.py (hunk 1 above)
Comment (rjuliang) [reply Flask-Integration/spellCheck.py:19]: Absolutely, we need to evaluate how to change those global variables from the Flask integration

File: Flask-Integration/get-localhost.py (hunk 2 above)
Comment (rjuliang) [reply Flask-Integration/get-localhost.py:13]: Sure, although I am not entirely familiar on what it does, I know CORS is a special permission that the API uses as a protocol to be accessible from the browser, we can do some research about it

File: Flask-Integration/shortTest.txt [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+mó
+in
+ilchrích
+Mheiriceá
+Theas
+atmaiféar
+cathacha
+príomhchahair
+habantrach
+réiiúin
```
Comment (Pream9) [inline Flask-Integration/shortTest.txt:109]: I think, we still more words for the Flask-Integration test for the Checking

File: Flask-Integration/spellCheck.py [hunk 4]
```diff
@@ -13,27 +13,32 @@
...
```
Comment (Pream9) [inline Flask-Integration/spellCheck.py:20]: Nice Change of Code @rjuliang to read the Irish txt as well.

File: Flask-Integration/unitTests.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+                INPUT_WORDS.append(word)
+    
+    with open('input_test_team6.tsv', 'w', encoding="UTF-8", newline='') as out_file:
+        tsv_writer = csv.writer(out_file, delimiter="\t")
+
+        for word in INPUT_WORDS:
+            correct = wordTestirish(word)
+            tsv_writer.writerow([word, correct])
+
+unitTests()
```
Comment (Pream9) [inline Flask-Integration/unitTests.py:27]: Neat Unit testing code, Nice Work. 
No changes needed

File: scripts/sendText.js [hunk 6]
```diff
@@ -10,9 +10,12 @@ function sendText(text){
 
     console.log('cursor position:',getCurrentCursorPosition('input'));
 
+    var baseUrlEndpoint = "http://localhost:5000/process?"
+
     //Third, we use the fetch method to call the localhost asynchronously (Synchronously was not recommended)
     //This is where we get the results from the index.js file
-    fetch("http://localhost:8080/spellCheck?lng="+languageSelected+"&text="+value, {
+    //fetch("http://localhost:8080/spellCheck?lng="+languageSelected+"&text="+value, {
+    fetch( baseUrlEndpoint+ "lng=" + languageSelected+"&phrase="+value, {
```
Comment (Pream9) [inline scripts/sendText.js:18]: Did we change the code to fetch the data text  without database
Or only by reading the text files, we uploaded?

File: Flask-Integration/shortTest.txt (hunk 3 above)
Comment (rjuliang) [reply Flask-Integration/shortTest.txt:109]: Pream, this is only a short version of the file sent by Dr. Scannell. It was used to see if the code was running as expected.

File: Flask-Integration/spellCheck.py (hunk 4 above)
Comment (rjuliang) [reply Flask-Integration/spellCheck.py:20]: Thanks Pream! :)

File: Flask-Integration/unitTests.py (hunk 5 above)
Comment (rjuliang) [reply Flask-Integration/unitTests.py:27]: Thanks!

File: scripts/sendText.js (hunk 6 above)
Comment (rjuliang) [reply scripts/sendText.js:18]: Pream,
Per project requirements, we need to develop a text editor. Therefore, we are not handling a file submission but we are handling user input and sending it to the Flask endpoint created by kimaya.
Let me know if you have questions.
Thank you,
Julian

**Pull request merged**